scripts/view.js

```diff
@@ -849,7 +849,7 @@ view.settings = {
 			// Enable based on image_overlay setting
 			if(!lychee.image_overlay) $('select#ImgOverlayType').attr('disabled', true);
 
-			$('select#ImgOverlayType').val(lychee.image_overlay_type === null ? '' : lychee.image_overlay_type);
+			$('select#ImgOverlayType').val(lychee.image_overlay_type === '' ? '' : lychee.image_overlay_type);
```

Comment:
here you need to use `lychee.image_overlay_type_default` instead. as this is the setting server side.